CONTRIBUTING.md

```diff
@@ -88,3 +88,58 @@ Every bug fix should be accompanied with a unit test, so that we can prevent reg
 ### Documentation, typos, ...
 
 They are mostly welcome!
+
+## Operator SDK
+
+Coming soon, but for the moment:
+
+Build the operator
+```
+BUNDLE_VERSION=0.16.0
+make set-image-controller IMG=quay.io/jpkroehling/opentelemetry-operator
```

Comment:
would it be better to replace your user id with something that obviously needs to be replaced? Or above set `export USER_ID=.....`  and then use `${USER_ID}` in the rest of the statements?

Reply:
I'll rewrite this section now that we have official confirmation that this is the right path.